src/autodiscovery.js

```diff
@@ -296,6 +298,7 @@ export class AutoDiscovery {
                 wellknown["m.identity_server"]["base_url"],
             );
             if (!isUrl) {
+                logger.log("[Debug] First error: isUrl is falsey");
```

Comment:
This seems unnecessary given that there's a `logger.error`statement right below?

Reply:
in theory, yes. There's about 2-3 places where we say `Invalid base_url for m.identity_server` on the error channel though :(